.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+
+**If you're here on behalf of another team looking to use this API, e.g. a side project or some other app, message `@Development` on the main Discord server and schedule a meeting with our backend team to discuss API use cases** so we're aware of and can plan for any incoming traffic from your app. And, when considering the entire tech architecture of our chapter, the features you're looking to build on top of the API in your app might better fit within the application boundaries of the portal itself; that's been the case for many API clients historically. Future teams can make use of that same functionality, and we can avoid disconnected apps and data siloes, as our membership data is deeply useful for building a higher-quality organization.
+
+**Please make sure to discuss your approach with a maintainer before starting any work** so that we can catch any major issues early. **Once you're ready to start coding, check out the README for instructions on getting the API running locally.** If any of our documentation is unclear or outdated, or you think our codebase could be doing something better, please suggest changes to make things easier for the next contributor!
+
+Once your functionality is implemented, open up a PR and a maintainer will review it as soon as possible. Anecdotally, most of our bigger PRs go through several rounds of review before merging, so if you're implementing a feature for an API client, be sure to start early, start often to meet your team's deadlines. If you're stuck or confused at any point, please don't hesitate to ask one of our maintainers for help.
+
+### Project Structure
+There're a lot of moving parts in this codebase and making simple changes can be daunting as a first-time contributor, so this section breaks down all the areas of the codebase you might need to edit to make your changes.
+
+#### API Specification
+You can find our API specification [here](https://documenter.getpostman.com/view/12949536/TVRedVwQ). It'll tell you about all the endpoints our API exposes and what each does. All the types in our specification, e.g. requests, responses, user profiles, merch orders, are defined in the `types` folder, and if you're adding a route or modifying an existing one, you'll first need to edit these types.
+
+#### API Implementation
+At a high level, our API is implemented in three layers: the endpoints layer, the services layer, and the data layer. The endpoints layer calls the services layer and the servies layer calls the data layer.
+
+##### Endpoints
+If you make a request to some endpoint, that request will be received in our endpoints layer, which handles anything that has to do with our routing logic, with requests and responses. This includes
```

Comment:
nit: "Endpoints" in rich-text mode looks smaller than the text below it (even if it's bolded). Maybe up all the header fonts by 1? Like you can just remove 1 # from every header